aws/data_source_aws_autoscaling_group_test.go

```diff
@@ -0,0 +1,99 @@
+package aws
+
+import (
+	"testing"
+
+	//"fmt"
```

Comment:
Commented imports should be removed. Tools like [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) generally have editor plugins which can automatically handle Go imports additions/removals/ordering during save. 😄